ci/packet_fluo/packet_fluo-cluster.lokocfg.envsubst

```diff
@@ -0,0 +1,97 @@
+cluster "packet" {
```

Comment:
How this will test FLUO? I think the plan was to use older Flatcar version?

Reply:
Those changes and tests to follow once the pipeline is enabled.